com.unity.netcode.adapter.utp/Editor/MultiplayerWindow.cs

```diff
@@ -0,0 +1,60 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.Netcode.Editor
+{
+    public class MultiplayerWindow : EditorWindow
```

Comment:
hmm, what's this?

Reply:
We added a quick editor for users to be able to use the Simulation Pipeline and tweak various settings about it

Reply:
As @Cosmin-B said this was at the request of the samples team, essentially DOTS Netcode has something similar